MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -4201,6 +4210,12 @@ public string GetVideoTransposeDirection(EncodingJobInfo state)
             if (doOclTonemap)
             {
                 var tonemapFilter = GetHwTonemapFilter(options, "opencl", "nv12");
+
+                if (isMjpegEncoder)
+                {
+                    tonemapFilter = tonemapFilter.Replace(":range=tv", ":range=pc", StringComparison.Ordinal);
```

Comment:
I more nasty way is to search for `:range` and if there is no such text we do add, otherwise replace. But I really don't think this function should be public in the first place. If an ABI change is not desired for now we have to do this I think.

Reply:
`range.auto` currently does nothing to the CLI, so `filter.Replace(":range=tv", string.Empty) + ":range=tv"` should be enough for now.

Reply:
But this results in `:range=pc:range=pc` even though it won't fail...

Reply:
We can search for `:range=pc` if we really want to. I really hope we can make all those method private to avoid doing things like this.

Reply:
If we want to avoid ABI changes in case we missed the ABI freeze deadline, we can make a `GetHwTonemapFilterwithRange` private method and do it there. But we still have like a few hours before the freeze, so this can be a last minute ABI change;

Reply:
This should be tolerable for bug fixing, i think.

BTW https://github.com/search?q=GetHwTonemapFilter&type=code